MekHQ/src/mekhq/campaign/personnel/SkillType.java

```diff
@@ -153,6 +138,82 @@ public static String getExperienceLevelName(int level) {
     private int eliteLvl;
     private Integer[] costs;
 
+   /**
+     * @param level skill level integer to get name for
+     * @return String skill name
+     */
+    public static String getExperienceLevelName(int level) {
+        return switch (level) {
+            case EXP_ULTRA_GREEN -> "Ultra-Green";
+            case EXP_GREEN -> "Green";
+            case EXP_REGULAR -> "Regular";
+            case EXP_VETERAN -> "Veteran";
+            case EXP_ELITE -> "Elite";
+            case -1 -> "Unknown";
+            default -> "Impossible";
+        };
+    }
+
+    /**
+     * @param level skill level integer to get color for
+     * @return String hex code for a font tag
+     */
+    public static String getExperienceLevelColor(int level) {
+        return switch (level) {
+            case EXP_ULTRA_GREEN -> MekHQ.getMHQOptions().getFontColorSkillUltraGreenHexColor();
+            case EXP_GREEN -> MekHQ.getMHQOptions().getFontColorSkillGreenHexColor();
+            case EXP_REGULAR -> MekHQ.getMHQOptions().getFontColorSkillRegularHexColor();
+            case EXP_VETERAN -> MekHQ.getMHQOptions().getFontColorSkillVeteranHexColor();
+            case EXP_ELITE -> MekHQ.getMHQOptions().getFontColorSkillEliteHexColor();
+            case -1 -> "";
```

Comment:
This will default to the default color based on the theme, correct?

Reply:
It should, because the functions that use it don't even generate a tag when the result is "".